contents/handbook/people/spending-money.md

```diff
@@ -12,9 +12,9 @@ Please spend money in a way that you think is in the best interests of the compa
 
 If it's a trivial expense, just buy it. We provide you with a company card with a $1,000/month spending limit for this reason. We use Brex for everyone, and also provide UK team members with a Revolut card. 
 
-You should use your Revolut card for UK-specific spending (i.e. ordering from UK sites, hotels/food/drink/... in the UK), and Brex for everything else. This is for UK accounts-reporting reasons, as we have a UK subsidiary.
+You should use your Revolut card for UK-specific spending (i.e. ordering from UK sites, hotels/food/drink/... in the UK), and Brex for everything else. This is for UK accounts-reporting reasons, as we have a UK subsidiary and can claim VAT back on many of those purchases.
 
-For larger expenses which don't fit into the items here, please **raise a policy suggestion for it as a pull request** in this doc, so we can document our decision making into our policy rather than making everything case by case.
+For larger expenses which don't fit into the items here, please _raise a policy suggestion for it as a pull request in this doc_, so we can document our decision making into our policy rather than making everything case by case.
```

Comment:
I would suggest clarifying that one-offs are okay. We don't want to create a policy change for every one-off case.